test/blocks/global-navigation/global-navigation.test.js

```diff
@@ -63,6 +64,49 @@ describe('global navigation', () => {
     });
   });
 
+  describe('Cross Clound Menu', () => {
```

Comment:
Nit: typo in 'Cloud'. Same for each `it` declaration

Reply:
Fixed 👍